cmd/sonicd/app/launcher.go

```diff
@@ -225,6 +225,7 @@ func lachesisMain(ctx *cli.Context) error {
 	if ctx.GlobalIsSet(config.FakeNetFlag.Name) {
 		cfg.OperaStore.EVM.StateDb.LiveCache = 1
 		cfg.OperaStore.EVM.StateDb.ArchiveCache = 1
+		cfg.OperaStore.EVM.Cache.StateDbCapacity = 1024
```

Comment:
I would favor an explicit flag for the state-db capacity that we set in our integration tests instead of limiting the cache size in general for fake-nets.

The reason is: Norma or other testing tools may employ fake-net setups also for scalability and/or performance tests. In those cases, the given capacity may be too small.